client/src/component/milestoneCard.css

```diff
@@ -1,37 +1,65 @@
 .milestone__card-container {
   display: flex;
   flex-direction: column;
-  border: 1px dotted black;
   width: 100%;
-  row-gap:30px
+  row-gap: 30px;
+  justify-content: center; /* Center horizontally */
+  align-items: center; /* Center vertically */
+  background-color: var(--PrimaryColor);
+  padding: 40px 15px;
+}
+.milestone__title-container {
+  display: flex;
+  flex-direction: column;
+  gap: 60px;
+}
+.milestone-title {
```

Comment:
@MohammadSefidgar was going to explain this to you all but the idea with the __ in the class names is to follow the BEM naming convention to avoid naming conflicts so the lasses here would be 
.milestone-card__title {}
.milestone-card__title-container {}